inc/Config/QueryRunner/QueryRunner.php

```diff
@@ -203,6 +203,11 @@ public function execute( HttpQueryInterface $query, array $input_variables ): ar
 			if ( ! array_key_exists( $key, $input_variables ) && isset( $schema['default_value'] ) ) {
 				$input_variables[ $key ] = $schema['default_value'];
 			}
+
+			// If the input variable is required and not provided, return an error.
+			if ( ! array_key_exists( $key, $input_variables ) && isset( $schema['required'] ) && $schema['required'] ) {
+				return new WP_Error( 'remote-data-blocks-missing-required-input-variable', sprintf( 'Missing required input variable: %s', $key ) );
```

Comment:
I'll be curious if this breaks any existing queries. I'd like to move towards full validation using our `Validator` class, but this will be a nice first step.